iceberg/patched-iceberg-core/pom.xml

```diff
@@ -95,7 +95,6 @@
                   <outputDirectory>${project.build.directory}/classes</outputDirectory>
                   <excludes>
                       **/HadoopInputFile.class
-                      **/SerializableTable.class
```

Comment:
Remove the patched class as this was fixed by upstream iceberg core codebase https://github.com/apache/iceberg/commit/07002fae27445408a4f8aeb512b14e1a555aa94f  by @deniskuzZ

Reply:
👍